crypto/cipher_extra/cipher_extra.c

```diff
@@ -106,21 +115,33 @@ const EVP_CIPHER *EVP_get_cipherbynid(int nid) {
   return NULL;
 }
 
+static const EVP_CIPHER *get_cipherbyname(const char* name) {
+  for (size_t i = 0; i < OPENSSL_ARRAY_SIZE(kCiphers); i++) {
+    if (OPENSSL_strcasecmp(kCiphers[i].name, name) == 0) {
+      return kCiphers[i].func();
+    }
+  }
+
+  return NULL;
+}
+
 const EVP_CIPHER *EVP_get_cipherbyname(const char *name) {
```

Comment:
I assume this is covered by the tcpdump integ test, but can you also add a unit test in our code.